src/sentry/integrations/discord/message_builder/base/base.py

```diff
@@ -17,7 +17,7 @@ class DiscordMessageBuilder(AbstractMessageBuilder):
 
```

Comment:
These changes are to support message overwriting. We need to have fields set to empty/nothing or else they are not overwritten when we try to update a message.